modules/filter_fastq_func.py

```diff
@@ -23,31 +24,63 @@ def check_params(**kwargs) -> dict:   # Функция проверяет вве
     quality_t = kwargs.get('quality_threshold')
     if quality_t is None:
         quality_t = 0
-    params1 = {'gc_bounds': (gc_b_low, gc_b_high), 'length_bounds': (lenght_b_low, lenght_b_high), 'quality_threshold': quality_t}
-    return params1
+    params = {'gc_bounds': (gc_b_low, gc_b_high), 'length_bounds': (lenght_b_low, lenght_b_high), 'quality_threshold': quality_t}
+    return params
+
+
+def fetch_seqs_from_file(input_file: str) -> dict:
+    "Read FASTQ file and fetch reads"
+    seqs = {}
+    with open(input_file) as file:
+        file_cont = file.read()
+    pre_seqs = file_cont.split('\n')
+    for i in range(0, len(pre_seqs) - 1, 4):
+        seqs[pre_seqs[i]] = (pre_seqs[i + 1], pre_seqs[i + 3])
+    return seqs
 
 
 def seqs_choose(seqs: dict, params: dict) -> dict:
     "Sequence choose by parameters"
     new_seqs = {}
     for name, seq in seqs.items():
-        gc_comp = gc_comp_calc(seq[0])
-        avg_quality = quality_calc(seq[1])
-        len_seq = len(seq[0])
-        if gc_comp <= params['gc_bounds'][1] and gc_comp >= params['gc_bounds'][0] and len_seq <= params['length_bounds'][1] and len_seq >= params['length_bounds'][0] and avg_quality >= params['quality_threshold']:
+        if check_gc(seq[0], params['gc_bounds']) and check_quality(seq[1], params['quality_threshold']) and check_lenght(seq[0], params['length_bounds']):
             new_seqs[name] = seq
     return new_seqs
 
 
-def gc_comp_calc(seq: str) -> float:
-    "Calculate GC composition of sequence (in%)"
-    return (seq.count('G')+seq.count('C'))/len(seq)*100
+def check_gc(seq: str, gc_comp: tuple) -> bool:
+    "Calculate GC composition of sequence (in%) and check if it in set parameters"
+    gc_content = (seq.count('G') + seq.count('C')) / len(seq) * 100
+    if gc_content <= gc_comp[1] and gc_content >= gc_comp[0]:
+        return True
+    return False
```

Comment:
никогда не нужно явно прописывать `return True` или `return False`
```suggestion
    return gc_content <= gc_comp[1] and gc_content >= gc_comp[0]
```

-1б